.gitignore

```diff
@@ -84,3 +81,16 @@ cpputest_*.xml
 #IAR automatically generated files
 *.dep
 *.ewt
+
+#CMake files
+CppUTestExtTests
+CppUTestTests
+Makefile
+CMakeFiles/
+CMakeCache.txt
+*.cmake
```

Comment:
`.cmake` are used within this project.

Reply:
I fixed the .gitignore to only ignore the generated files.